auguste/roster.json

```diff
@@ -74,5 +74,9 @@
       "name": "Nadir Khan",
       "github": "Nadirkhan98"
     }
+    {
+      "name": "William Sessions",
+      "github": "Izgardon"
+    }
```

Comment:
Can you please add a comma between the objects?
    },  <<--- comma needed :) 
    {
      "name": "William Sessions",
      "github": "Izgardon"
    }